internal/provider/resource_ssh_key.go

```diff
@@ -129,9 +132,9 @@ func (s *sshKeyResource) Create(ctx context.Context, req resource.CreateRequest,
 		resp.Diagnostics.AddError("Client Error", fmt.Sprintf("Unable to create ssh_key, got error %s", err))
 		return
 	}
-	s.trace(fmt.Sprintf("created ssh_key for: %q", keyIdentifier))
+	s.trace(fmt.Sprintf("created ssh_key for: %q", fingerprint))
 
-	plan.ID = types.StringValue(newSSHKeyID(modelName, keyIdentifier))
+	plan.ID = types.StringValue(newSSHKeyID(modelName, fingerprint))
```

Comment:
Will this not impact users with existing state? Or you're thinking the delete logic supports delete by comment or fingerprint so it's fine?